specs/ThrottlingControlScript.md

```diff
@@ -0,0 +1,325 @@
+Throttling Control - Script Throttling
+===
+
+# Background
+Web content in WebView2 is generally subject to the same Web Platform
+restrictions as in the Microsoft Edge browser. However, some of the scenarios
+for WebView2 applications differ from the scenarios in the browser. For this
+reason, we're providing a set of APIs to fine-tune performance of scripts
+running in WebView2. These APIs allow WebView2 applications to achieve two
+things:
+
+* Customize script timers (`setTimeout` and `setInterval`) throttling under
+different page states (foreground, background, and background with intensive
+throttling)
+* Throttle script timers in select hosted iframes
+
+# Examples
+
+## Throttle timers in visible WebView
+
+Throttling Control APIs allow you to throttle JavaScript timers in scenarios
+where the WebView2 control in your application needs to remain visible, but
+consume less resources, for example, when the user is not interactive.
+
+```c#
+void OnNoUserInteraction()
+{
+    // User is not interactive, keep webview visible but throttle timers to 500ms.
+    webView.CoreWebView2.Settings.ThrottlingIntervalPreferenceForeground = 500;
```

Comment:
Do we have precedence for wv2 intervals implicitly being milliseconds?  I was going to propose including this in the property names, but realized we're aligning to JS' global functions that already do this.  I didn't quickly find an existing webview2-related milliseconds property to check.

(Closest was [bufferUsageReportingInterval](https://learn.microsoft.com/en-us/microsoft-edge/webdriver-chromium/capabilities-edge-options), so maybe this is OK?)

Reply:
See comment above on "Milliseconds": https://github.com/MicrosoftEdge/WebView2Feedback/pull/4156#discussion_r1401102994